modules/application/application.js

```diff
@@ -32,6 +32,90 @@ class Application {
         checkList.checked = true;
       }
     });
+
+    this.eventDispatcher();
+  }
+
+  eventDispatcher = () => {
+    this.onclickeventDispatcher();
+    this.onsubmiteventDispatcher();
+    this.onfocusouteventDispatcher();
+  }
+
+  onclickeventDispatcher = () => {
+    const buttons = document.querySelectorAll('.delete');
+    const ref = this;
+    buttons.forEach((button, index) => {
+      button.addEventListener('click', (event) => {
+        const eventIdentifier = event.currentTarget;
+        const localArray = [];
+        let count = 0;
+        for (let i = 0; i < eventIdentifier.ref.todoList.length; i += 1) {
+          if (i !== eventIdentifier.index) {
+            eventIdentifier.ref.todoList[i].index = count;
+            localArray.push(eventIdentifier.ref.todoList[i]);
+            count += 1;
```

Comment:
Please ensure that the index of items starts from `1` to make it user-friendly.

Reply:
User friendly in what way? didn't see this in the requirement but I'd make changes

Reply:
@harlexkhal,
![image](https://user-images.githubusercontent.com/10332499/154394351-7781fa70-e151-478c-bac4-5bdc61af40b1.png)

Reply:
Oh, Alright. Thanks